src/content/docs/contributing/nf_core_basic_training/add_custom_module.md

```diff
@@ -301,3 +301,12 @@ In the directory `exercise_6` you will find the custom script `print_hello.py`,
       </details>
 
 :::
+
+<p class="text-center">
+  <a href="../add_nf_core_module/" class="btn btn-lg btn-success" style="font-size: 14px">
```

Comment:
```suggestion
  <a href="/docs/contributing/add_nf_core_module/" class="btn btn-lg btn-success" style="font-size: 14px">
```
and so on 🙂

Reply:
what are the local links pointing to? and do you get a redirect message in the terminal?

Reply:
Now both local and remote seems to work. idk. I did previously get a redirect. But since it is working now, I am not going to further investigate. Thank @mashehu !